application/lib/console.dart

```diff
@@ -111,7 +119,45 @@ class _ConsoleState extends State<Console> {
               _buildChip('take all'),
               _buildChip('drop all'),
               _buildChip('debug status'),
-              _buildChip('debug locations'),
+              OutlinedButton(
+                child: const Text('Teleport'),
+                onPressed: () {
+                  widget.game
```

Comment:
we should make this logic handle the case where it's called while already running, in some reasonable way that doesn't lead to multiple dialogs

could be to have the async logic in the dialog, with a spinner, or could be to have a lock boolean that ignores additional taps, or could be to disable the button while it's busy, or...